patientsearch/api.py

```diff
@@ -596,3 +596,16 @@ def main():
         "home.html",
         cache_timeout=-1,
     )
+
+
+@api_blueprint.route("/target", methods=["GET"])
+@oidc.require_login
+def target():
+    return send_from_directory(
```

Comment:
@achen2401 I believe you'll need to capture and pass the query string parameters, or did you find them still available on the request?

Reply:
@pbugni I find that query string parameters still available in the request.  This is true when I tried to access the link from the email.